js/controllers.js

```diff
@@ -48,17 +51,21 @@
             Forks.query({user: $scope.user, repo: $scope.repo, page: $scope.currentPage, perPage: $scope.perPage},
                     function(data) {  // query method expects array data
                         $scope.forks = data;
-                        $scope.noRepo = false;
-                        $scope.noFork = false;
+                        $scope.showMessage = false;
                         if (data.length == 0) {
-                            $scope.noFork = true;
+                            $scope.showMessage = true;
```

Comment:
Nice refactoring! I was wondering if there would be a way to not having to set $scope.showMessage = true; or false, but just setting $scope.message, so that if there is no message then in the view it is hidden and shown the messages if they are set...